Graphics/AIGraphics/AIGraphics.csproj

```diff
@@ -32,35 +32,42 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="0Harmony, Version=2.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\IllusionLibs.BepInEx.Harmony.2.0.3.1\lib\net35\0Harmony.dll</HintPath>
+      <HintPath>..\..\..\..\illusion\ais\BepInEx\core\0Harmony.dll</HintPath>
```

Comment:
Please don't change the reference path. We're using the NuGet Package to deal with this problem.